### PR TITLE
Add localization for Belarusian language

### PR DESCRIPTION
**Pull request by MaximFedarau**
I added support for Belarusian language, because community of Belarusian-speaking programmers is fast growing.

### CODE REVIEW

File: localization/languages/be.json [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+      "settingsCustomBangsPhrase": "Фраза (Абавязкова)",
+      "settingsCustomBangsSnippet": "Апісанне (Неабавязкова)",
+      "settingsCustomBangsRedirect": "URL перанакіравання (Абавязкова)",
+      "settingsCustomizeFiltersLink": "Настроить фильтры",
+      "settingsAppearanceHeading": "Знешні выгляд",
+      "settingsEnableDarkMode": "Выкарыстаць цёмную тэму:",
+      "settingsDarkModeNever": "Ніколі",
+      "settingsDarkModeNight": "Толькі ноччу",
+      "settingsDarkModeAlways": "Заўсёды",
+      "settingsDarkModeSystem": null,
```
Comment (PalmerAL) [inline localization/languages/be.json:152]: Can you add a translation for this?

Comment (MaximFedarau) [reply localization/languages/be.json:152]: Ok. One moment.

**Comment by MaximFedarau** [comment]
Commited this change.

**Comment by PalmerAL** [comment]
Thanks!

**Pull request merged**